src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -251,65 +254,66 @@ export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
       key: this.regKey,
       moderatedPerson: this.moderatedPerson,
       previewAs: this.previewAsPerson,
-    }).subscribe((feedbackSession: FeedbackSession) => {
-      this.feedbackSessionInstructions = feedbackSession.instructions;
-      this.formattedSessionOpeningTime = this.timezoneService
+    }).pipe(finalize(() => this.hasFeedbackSessionLoaded = true))
```

Comment:
Just a minor thing:

Would `isFeedbackSessionLoading` be a better name? When i did the indicators, the variable naming for loading i adopted was `isSomethingLoading` since the directive is `tmIsLoading`